node-repository/src/main/java/com/yahoo/vespa/hosted/provision/autoscale/QuestMetricsDb.java

```diff
@@ -152,7 +154,8 @@ private void ensureExists(String tableName) {
 
         try (SqlCompiler compiler = new SqlCompiler(engine)) {
             compiler.compile("create table " + tableName +
-                             " (hostname string, at timestamp, cpu_util float, mem_total_util float, disk_util float, application_generation long)" +
+                             " (hostname string, at timestamp, cpu_util float, mem_total_util float, disk_util float," +
```

Comment:
What about the existing tables?

Reply:
They will get default values, which is not what we want here (we want true not false), but the only effect is no autoscaling for a while, so I think that's okay.

Reply:
So as soon as you add a row with 6th column, that is legal even though the table was created with 5 columns, and all existing rows will get a default value of the type of the 6th column in the new row?

Reply:
Apparently - I tested it. Couldn't find any documentation on this scenario.